test/verify/check-connection

```diff
@@ -529,6 +529,20 @@ G_MESSAGES_DEBUG=all XDG_CONFIG_DIRS=/usr/local %s -p 9999 -a 127.0.0.90 --local
         self.assertIn("HTTP/1.1 301 Moved Permanently", m.execute("curl --silent --head http://172.27.0.15:9091"))
         self.assertIn("HTTP/1.1 200 OK", m.execute("curl --silent --head http://127.0.0.1:9091"))
 
+    def testCaCert(self):
+        m = self.machine
+        b = self.browser
+
+        m.start_cockpit()
+
+        # The images don't have sscg, so we'll don't get a CA certificate.
```

Comment:
They actually do. But the command below removes it. Please just drop the comment.

Reply:
Done.